tests/data/hdfs/hadoop.env

```diff
@@ -0,0 +1,43 @@
+CORE_CONF_fs_defaultFS=hdfs://namenode:9000
```

Comment:
**note**: This was totally just taken from [here](https://github.com/big-data-europe/docker-hadoop/blob/master/hadoop.env) I don't know which options here are actually important for this test 😅 

It could be worth trimming down to the bare minimums to avoid confusion in the future, if anyone knows or wants to dive into it and figure that out.

Reply:
I'm guessing we can remove all yarn and mapreduce related conf.

Reply:
I'll test that locally and push it up if things still pass - I'll also fix the few problems reported by clippy.

Reply:
Thanks a lot!

Reply:
😢 and I can't get the integration tests running properly locally again. HDFS doesn't seem to be starting up (without having changed any of the config).

Is there a way we could do the integration test without having to pull in the HDFS client, maybe with just a raw html client instead? Pulling in java/hdfs for generic Vector hacking is a little painful (running `cargo clippy --workspace --all-targets --features all-integration-tests -- -D warnings` fails because we don't have those crate deps.

Reply:
> Is there a way we could do the integration test without having to pull in the HDFS client, maybe with just a raw html client instead?

OpenDAL supports [webhdfs](https://opendal.databend.rs/opendal/services/struct.Webhdfs.html) which is pure http based and doesn't need hdfs (java) client. But it's different from hdfs and requires user to enable webhdfs feature on their hdfs deployment. They should be treated as two seperated sinks.

> running cargo clippy --workspace --all-targets --features all-integration-tests -- -D warnings fails because we don't have those crate deps.

To run clippy on `all-integration-tests` (include hdfs-integration-tests), we only need jre (to link to `jni.h`) instead of a full hadoop runtime which should already be set up if we have pulsar and other java related integrations.

Reply:
Wow, new versions of venv seems added podman support...